libgstc/pygstc/gstc.py

```diff
@@ -182,7 +182,8 @@ def _check_parameters(self, parameter_list, type_list):
             if not isinstance(parameter, type_list[i]):
                 raise GstcError(
                     "%s TypeError: parameter %i: expected %s, '%s found" %
-                    (inspect.stack()[1].function, i, type_list[i], type(parameter)))
+                    (inspect.stack()[1].function, i, type_list[i],
+                     type(parameter)), -6)
```

Comment:
What is this number (-6) ?

Reply:
Agreed, do not hardcode this. Add this enum to the gstcerror module.